src/include/migraphx/op/squeeze.hpp

```diff
@@ -37,22 +37,30 @@ struct squeeze
     std::string name() const { return "squeeze"; }
     shape normalize_compute_shape(std::vector<shape> inputs) const
     {
-        check_shapes{inputs, *this}.has(1).standard();
+        check_shapes{inputs, *this}.has(1);
         auto input_shape = inputs[0];
         auto type        = input_shape.type();
         auto old_lens    = input_shape.lens();
-
+        auto old_strides = input_shape.strides();
         if(std::any_of(axes.begin(), axes.end(), [&](auto axis) { return old_lens[axis] != 1; }))
         {
             MIGRAPHX_THROW("squeeze axis dimension should be equal to 1");
         }
         std::vector<std::size_t> new_lens;
+        std::vector<std::size_t> new_strides;
         if(axes.empty())
         {
             std::copy_if(old_lens.begin(),
                          old_lens.end(),
                          std::back_inserter(new_lens),
                          [](auto len) { return len != 1; });
+            for(std::size_t i = 0; i < old_lens.size(); i++)
+            {
+                if(old_lens[i] != 1)
+                {
+                    new_strides.push_back(old_strides[i]);
+                }
```

Comment:
The `copy_if` from above could be combined with this loop so you would just do:

```cpp
                if(old_lens[i] != 1)
                {
                    new_strides.push_back(old_strides[i]);
                    new_lens.push_back(old_lens[i]);
                }
```

Reply:
Done thanks.